ui/components/Dependencies.jsx

```diff
@@ -52,12 +52,14 @@ export default connect(
       packageSelectors.getSelectedPackageName(state.packages),
     ),
   }),
-  dispatch =>
-    bindActionCreators(
-      {
-        onDependencyClick: actions.removeDependency,
-        onAddClick: actions.addDependency,
-      },
-      dispatch,
-    ),
+  dispatch => ({
+    onAddClick: name => {
+      dispatch(uiActions.showTerminal());
+      dispatch(actions.addDependency(name));
+    },
+    onDependencyClick: name => {
```

Comment:
`onRemoveClick` is a better name I guess?

Reply:
Yes, now i change it